website/source/docs/configuration/client.html.md

```diff
@@ -349,6 +352,31 @@ see the [drivers documentation](/docs/drivers/index.html).
   files on the client host via the `file` function. By default templates can
   access files only within the task directory.
 
+### `host_volume` Stanza
+
+The `host_volume` stanza is used to make volumes available to jobs.
+
+The key of the stanza corresponds to the name of the volume for use in the 
+`source` parameter of a `"host"` type [`volume`](/docs/job-specification/volume.html)
+and ACLs.
+
+```hcl
+client {
+  host_volume "ca-certificates" {
+    path = "/etc/ssl/certs"
+    read_only = true
```

Comment:
will this make `read_only` job parameter irrelevant? if the job says `read_only=false` it would still be read-only by client config? or will it be a constraint saying you need this volume with this read_only value?

Reply:
Oh I should clarify this - if the host_volume is read_only=true, then only groups with read_only=true will be scheduled on it

Reply:
so the cluster can have two `host_volume "ca-certificates" {` where one is read-only and one is read-write, and nomad will find the right one based on all the fields as a constraint?

Can I have two `host_volume "ca-certificates" {` in the same client config having a different path and/or read_only flag?

Reply:
Host Volumes require unique names on a client, (e.g client has a map[Name]Config), but multiple clients may have volumes with the same name.

If the host_volume is `read_only = false` then any request for that volume may be scheduled, even if the volume request is `read_only = true`. Setting the host_volume to read_only will however prevent a `read_only = false` request from being scheduled on it.

If you wish to restrict developers from using a volume in write mode, but want operational tooling to have write access, then you can declare it `read_only = false`, and then use the ACL system to only give developers `mount-readonly` permissions for that volume, and `mount-readwrite` permissions to your ops tokens.

Reply:
@endocrimes thanks for explaining it :)